### ISSUE TITLE
Create Flux daemon to serve v2 API backed by InfluxDBv3

### ISSUE THREAD

**Issue by pauldix** [influxdata/influxdb#24375]
This issue is a catchall for the moment to track interest and community efforts to update Flux to use InfluxDB 3.0 as a back end. We anticipate that the best way to do this will be to update the core Flux engine to use 3.0's FlightSQL API. This is likely not a small effort and will require changes to how the Flux planner and optimizer work.

The [Flux repo is here under a permissive MIT license](https://github.com/influxdata/flux). We've also created a [community Flux repo](https://github.com/InfluxCommunity/flux) that can be owned by the community for this effort. We think this will enable people to move more quickly without having to push things through our production code process (which would happen for anything landing in the main Flux repo).

**Comment by pauldix** [comment]
Yes definitely, Fluxpipe looks promising as a path forward for everyone. Please feel free to keep this thread updated with any progress you think is worth mentioning.

**Comment by lmangani** [comment]
For lack of better options we are also hosting the [Flux Community fork documentation](https://metrico.github.io/fluXpipe/#/) generated by community fluxlib

**Comment by dolfno** [comment]
It would save quite some functionality loss and migration efforts. 
We use flux in Grafana and influx tasks, because it allows quite some business logic in the queries.
e.g. calculate energy mass balance by dividing the time weighted values of energy consumption with production throughput. (which is like the integral(energy) / integral(production) for a given time interval)

**Comment by mdealer** [comment]
Yes, the flexibility that Flux provided can only be replicated by a pipeline within an infrastructure conglomerate of multiple DBs and query languages. This should not be an initial requirement for a random project in 2023 anymore... On our side we track performance of our test system. The advanced filtering that we can do based on tags alone is gold. It is not an efficient solution for time series DBs, but the well organized docs and tons of extensible functions within Flux is a win-win-win for some small to mid sized scenario even given some performance issues of 2.x.
